web/src/app/schedules/ScheduleOverrideForm.js

```diff
@@ -92,13 +96,15 @@ export default function ScheduleOverrideForm(props) {
             .
           </Typography>
         </Grid>
-        <Grid item xs={12} sm={12} md={6}>
-          {/* Purposefully leaving out of form, as it's only used for converting display times. */}
-          <ScheduleTZFilter
-            label={(tz) => `Configure in ${tz}`}
-            scheduleID={scheduleID}
-          />
-        </Grid>
+        {!isLocalTZ && (
+          <Grid item xs={12} sm={12} md={6}>
+            {/* Purposefully leaving out of form, as it's only used for converting display times. */}
+            <ScheduleTZFilter
+              label={(tz) => `Configure in ${tz}`}
+              scheduleID={scheduleID}
+            />
+          </Grid>
+        )}
```

Comment:
This logic needs to be inside `ScheduleTZFilter`, I think what you intended was just a check if `schedule.timeZone === DateTime.local().zone.name`

Reply:
Will continue in issue